pyproject.toml

```
@@ -33,7 +33,7 @@ dependencies = [
 "sentencepiece>=0.1.99,<0.3",
 "tokenizers>=0.13.3,<1.0",
 "tqdm>=4.66.2,<5.0",
-"trl>=0.9.3,<0.12",
+"trl>=0.9.3,<1.0",
```

@willmj do we want to probably lock the version to higher than one which supports packing for pretokenized datasets which we enabled along with this patch.

---

@dushyantbehl I thought the latest version was 0.13.0, so this would include that. Feel free to make a suggested change if I'm misunderstanding.

---

```suggestion
"trl>=0.13.0,<1.0",
```

@dushyantbehl is this your suggestion?